owncloud/core
# Add calendar sharing

# Test Scenario
1. create a calendar (./occ dav:create-calendar user shared-calendar)
2. POST a sharing xml request to the calendar url https://github.com/owncloud/core/blob/calendar-sharing-3/apps/dav/tests/travis/caldavtest/data/Resource/CalDAV/sharing/calendars/read-write/1.xml
3. see the calendar popping up with the other user

You can use cadaver to check for the calendar resources.
